.taskcluster.yml

```diff
@@ -24,9 +24,15 @@ tasks:
           go get -t github.com/taskcluster/taskcluster-cli/... 
           && cd  /go/src/github.com/taskcluster/taskcluster-cli 
           && git fetch {{ event.head.repo.url }} {{ event.head.ref }} 
-          && git checkout {{ event.head.sha }} 
           && make 
-          && go test ./...
+          && go get -u github.com/alecthomas/gometalinter 
+          && gometalinter --install
+          && go get -u ./...
+          && git checkout {{ event.head.sha }} 
```

Comment:
This is to make `gometalinter` happy, as it doesn't look for packages inside vendor. Unfortunately, there is no much more we can do about it, since if we try to do `go get` after `git checkout`, it will fail due to the repo being in a detached state.